### PR TITLE
feat(config): add Zooz ZEN04

### PR DESCRIPTION
**Pull request by mikebaz**
This change adds a configuration file for the ZEN04 smart plugs.  Values are based on the ZEN05 file and the info at https://www.support.getzooz.com/kb/article/1114-zen04-smart-plug-advanced-settings/.

### CODE REVIEW

File: packages/config/config/devices/0x027a/zen04.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+				{
+					"label": "Always off",
+					"value": 1
+				},
+				{
+					"label": "Always on",
+					"value": 2
+				}
+			]
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen04.json]: This should use a template like the ZEN05 does here:
https://github.com/zwave-js/node-zwave-js/blob/81d6306c505b8af4d51ba8a10e645c6a8dd0f7a7/packages/config/config/devices/0x027a/zen05.json#L48-L51
except that you need the `..._prev_off_on` variant.

File: packages/config/config/devices/0x027a/zen04.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+				},
+				{
+					"label": "Always off",
+					"value": 2
+				},
+				{
+					"label": "Always on",
+					"value": 3
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen04.json]: Please change to this for consistency:
```suggestion
		"label": "LED Indicator",
		"valueSize": 1,
		"defaultValue": 0,
		"allowManualEntry": false,
		"options": [
			{
				"label": "On when load is on",
				"value": 0
			},
			{
				"label": "On when load is off",
				"value": 1
			},
			{
				"label": "Always off",
				"value": 2
			},
			{
				"label": "Always on",
				"value": 3
			}
		]
```

File: packages/config/config/devices/0x027a/zen04.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		{
+			"#": "2",
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration"
+		},
+		{
+			"#": "3",
+			"$import": "templates/zooz_template.json#auto_on_timer_duration",
+			"label": "Auto Turn-On Timer Duration"
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen04.json]: These don't need the label specified again, it comes from the template

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, just a few remarks

File: packages/config/config/devices/0x027a/zen04.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x027a/zen04.json]: (looks like I messed up the indentation, but the bot will correct that)

**Comment by mikebaz** [comment]
ok, I believe I correctly implemented the requested changes.  Sorry for not getting it right the first time - I know that wastes time on your end. :(

**Comment by AlCalzone** [comment]
All good - this was one of the better PRs ;)
Our standards are pretty high, so it's not easy to fulfill them.

**Review by AlCalzone — APPROVED** [review]

**Comment by gminco** [comment]
Could we be missing one extra parameter?
I see 8 on the Zooz site but after including one I'm counting 9:
image

**Comment by gminco** [comment]
We're missing parameter #9, I've tried but I cannot edit the file, all it needs is this to be added:

`,
		{
			"#": "9",
			"$import": "templates/zooz_template.json#led_indicator_brightness"
		}`

**Comment by AlCalzone** [comment]
@gminco I cannot find parameter 9 in [Zooz's documentation](https://www.support.getzooz.com/kb/article/1114-zen04-smart-plug-advanced-settings/). If it is actually missing, please open a new issue and share the manual or whatever states that it does exist.

**Comment by gminco** [comment]
@AlCalzone it's on the documentation link you provided, but it's between parameter 1 and 2.

**Comment by mikebaz** [comment]
@gminco ah you're right, I missed that.  as mentioned the update requires a new PR.  I'll get one in shortly.

**Pull request merged**